tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -120,9 +120,19 @@ def __init__(self):
         self.kernel_list_groupby_name_op: Dict[str, KernelAggByNameOp] = None  # For Kernel-view.
         self.stack_lists_group_by_name: Dict[str, List[OperatorAgg]] = None
         self.stack_lists_group_by_name_input: Dict[str, List[OperatorAgg]] = None
-        self.tc_eligible_ops = None
+        self.tc_eligible_ops_kernel_ratio = 0.0
 
     def aggregate(self, tid2tree):
+        def sum_self_kernel_time(ops):
+            sum_time = 0
+            for op in ops:
+                for rt in op.runtimes:
+                    # "CallTreeRoot" & "dummy" kernels are launched out of profiler step, so don't count them.
+                    if op.name != "CallTreeRoot" and rt.name != "dummy":
```

Comment:
the kernels under dummy node will be ignored?

Reply:
Yes. Because we can't get which operators launch those kernels of dummy node, so we can't know these kernels are launched by TC-eligible ops or none-TC-eligible ops.